src/id3manager/formats/formats.py

```diff
@@ -0,0 +1,26 @@
+import typing as t
+
+from .abc import MetadataFormatter
+from .text import TextMetadataFormatter
+from .toml import TomlMetadataFormatter
+
+
+FORMATTERS = {
+    formatter.name(): formatter
+    for formatter in [TextMetadataFormatter(), TomlMetadataFormatter()]
```

Comment:
nit: I think it's fine to have global instances for now, but in the future I can imagine the formatters to receive formatter specific settings. E.g. encoding, delimiters/quotechar (in case of some CSV), etc.

Reply:
Yeah, that's an interesting idea! I guess we would need to add some sort of dependency injection and instantiate the instances from `main()` where we have access to the command line arguments.